compiler/ballerina-parser/src/test/resources/expressions/xml-template/xml_template_source_32.bal

```diff
@@ -0,0 +1,11 @@
+function foo() {
+    xml a = xml `<![CDATA[first text ]]> second text ]]>`;
+
+    xml b = xml `<![CDATA[first text `;
+
+    xml c = xml `<<![CDATA[>]]>> xml content <<![CDATA[>]]>/>`;
+
+    xml d = xml `]]>`;
```

Comment:
this one also?

Reply:
The first test is having CDATA Sections as XML element names which is invalid. Idea of second test is to check CDATA Section with missing start token but it will detect it as valid XML text.